app/models/manageiq/providers/kubevirt/infra_manager/vm/operations.rb

```diff
@@ -5,6 +5,7 @@ module ManageIQ::Providers::Kubevirt::InfraManager::Vm::Operations
   include Power
   include Snapshot
   include Guest
+  include InfraVolume
```

Comment:
Minor, but in terms of naming I think `Disk` would be a better module name as that is in line with our nomenclature